exercises/high-scores/high_scores.py

```diff
@@ -1,3 +1,10 @@
-class HighScores(object):
-    def __init__(self, scores):
-        pass
+def latest(scores):
+    pass  # Put your code here
```

Comment:
Stub files for other exercises don't include this comment, and since this isn't the first exercise, students should be familiar with how the stub works by now.

```suggestion
    pass
```